lib/ansible/modules/windows/win_dns_record.py

```diff
@@ -76,6 +77,13 @@
     value: "cgyl1404p.amer.example.com"
     zone: "amer.example.com"
 
+- name: PTR example
+  win_dns_record:
+    name: "db1"
```

Comment:
@ShachafGoldstein I think that you reversed the `name` and the `value` parameters here:
It should be:
```
name: "1.1.1"
```